examples/2d/mesh2d_arcs.rs

```diff
@@ -70,7 +71,7 @@ fn setup(
             Mesh2d(meshes.add(sector_mesh)),
             MeshMaterial2d(material.clone()),
             Transform {
-                translation: Vec3::new(FIRST_X + OFFSET * i as f32, 2.0 * UPPER_Y, 0.0),
+                translation: Vec3::new(FIRST_X + OFFSET * i as f32, FIRST_Y + 2.0 * UPPER_Y, 0.0),
```

Comment:
It seems that these are now centered horizontally, which is great, but they've become un-centered vertically.